server/routes/induction/common/qualificationsListController.ts

```diff
@@ -37,6 +37,7 @@ export default abstract class QualificationsListController extends InductionCont
   }
 }
 
+// TODO - this is duplicated in WantToAddQualificationsController - needs putting somewhere common
```

Comment:
Yeah; agreed. I wonder if the router request handler is the right place to do this (or perhaps a specific router request handler that returns just the most recent assessments .... 🤔